Tests/NIOHTTP2Tests/SimpleClientServerInlineStreamMultiplexerTests.swift

```diff
@@ -329,12 +329,55 @@ class SimpleClientServerInlineStreamMultiplexerTests: XCTestCase {
         stream.writeAndFlush(headerPayload, promise: promise)
     }
 
-    func testSuccessfullyReceiveAndSendPingEvenWhenConnectionIsFullyQuiesced() throws {
+    func testOpenStreamBeforeReceivingAlreadySentGoAway() throws {
         let serverHandler = InboundFramePayloadRecorder()
         try self.basicHTTP2Connection() { channel in
             return channel.pipeline.addHandler(serverHandler)
         }
 
+        let clientHandler = InboundFramePayloadRecorder()
+        let childChannelPromise = self.clientChannel.eventLoop.makePromise(of: Channel.self)
+        let multiplexer = try (self.clientChannel.pipeline.context(handlerType: NIOHTTP2Handler.self).wait().handler as! NIOHTTP2Handler).multiplexer.wait()
```

Comment:
This can be simplified a lot (I'm guessing the tests were written before this API was added) by using the `handler(type:)` API (see https://swiftpackageindex.com/apple/swift-nio/2.70.0/documentation/niocore/channelpipeline/handler(type:))